pages/add/form/detail-info-fields.js

```diff
@@ -80,15 +51,17 @@ let Tags = React.createClass({
     }).filter(suggestion => !!suggestion);
   },
   render: function() {
+    let delimiters = [9,13,32,188]; // keycodes for tab,enter,space,comma
```

Comment:
Interesting point, I had a look and `keyCode` is apparently obsolete. I've filed this over on https://github.com/i-like-robots/react-tags/issues/81 too.

To resolve this rather than `charCodeAt(0)` we can use `codePointAt(0)` which yields the correct value.